src/actions/channels.ts

```diff
@@ -527,6 +528,9 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
             {
                 type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBERS,
                 data: channelMembers,
+                sync: !shouldFetchArchived,
+                channels,
+                serverVersion: getServerVersion(getState()),
```

Comment:
do we still need to pass this to the reducer, if so why? the same with the `remove` property

Reply:
`remove` was introduced [here](https://github.com/mattermost/mattermost-redux/commit/019dd7625b9f3960371829d0e075aed6e5ce8a1a) and I thought it was already needed. I can't go back to the initial reasoning to include the remove there is only a commit for that change without a ticket.